packages/twenty-server/src/engine/core-modules/workspace/workspace.resolver.ts

```diff
@@ -387,4 +386,12 @@ export class WorkspaceResolver {
       workspaceGraphqlApiExceptionHandler(err);
     }
   }
+
+  @Mutation(() => DomainValidRecords, { nullable: true })
+  @UseGuards(WorkspaceAuthGuard)
+  async checkCustomDomainValidRecords(
```

Comment:
Reading this made me want to create this: https://github.com/twentyhq/twenty/issues/14405